### PR TITLE
Plugin upgrade

### PR DESCRIPTION
**Pull request by andersonmmi**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,7 +9,7 @@
 /coverage
 
 # production
-/build
+#/build
```
Comment (rpavlovs) [inline .gitignore]: why?

File: build/asset-manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (rpavlovs) [inline build/asset-manifest.json]: This shouldn't be checked in

File: src/exchange-bldg.js [hunk 3]
```diff
@@ -1,58 +1,58 @@
...
+import SendPage from "@burner-wallet/ui/dist/Pages/SendPage"
+const URLREGEX = /payment\/(0x[0-9a-f]{40})\/(\w*)\/([\d\.]+)\/(\w*)/i;
 
 export default class ExchangeBldg {
+  constructor() {
+    // this.state = {
+    //   asset:"it",
+    //   to:'0xe8bF424E047372d249d0826c5567655ba3B72f18'
+    // }
+  }
```
Comment (rpavlovs) [inline src/exchange-bldg.js]: 💣

File: src/exchange-bldg.js [hunk 4]
```diff
@@ -1,58 +1,58 @@
...
+
   initializePlugin(pluginContext) {
+    this._pluginContext = pluginContext;
+
+    // pluginContext.addElement('home-top', Selector)
+    // pluginContext.addPage('/pay', SendPage)
```
Comment (rpavlovs) [inline src/exchange-bldg.js]: 💣

File: src/index.tsx [hunk 5]
```diff
@@ -43,15 +43,15 @@ const core = new BurnerCore({
...
-  assets: [xdai, exchangeBldg, brass, it]
+  assets: [exchangeBldg, brass, it, xdai]
 });
 
-const exchange = new Exchange({
-  pairs: [xdaiBridge, uniswapDai]
-});
+// const exchange = new Exchange({
+//   pairs: [xdaiBridge, uniswapDai]
+// });
```
Comment (rpavlovs) [inline src/index.tsx]: 💣

File: src/index.tsx [hunk 6]
```diff
@@ -43,15 +43,15 @@ const core = new BurnerCore({
...
 
 const BurnerWallet = () => (
-  <BurnerUI core={core} plugins={[exchange, new ExchangeBldg()]} />
+  <BurnerUI core={core} plugins={[ new ExchangeBldg()]} />
```
Comment (rpavlovs) [inline src/index.tsx]: no space ✨

File: .gitignore (hunk 1 above)
Comment (andersonmmi) [reply .gitignore]: no clue

File: build/asset-manifest.json (hunk 2 above)
Comment (andersonmmi) [reply build/asset-manifest.json]: :boom:

File: .gitignore (hunk 1 above)
Comment (rpavlovs) [reply .gitignore]: this should stay in gitignore

File: src/exchange-bldg.js (hunk 3 above)
Comment (andersonmmi) [reply src/exchange-bldg.js]: :boom:

File: src/exchange-bldg.js (hunk 4 above)
Comment (andersonmmi) [reply src/exchange-bldg.js]: :boom:

File: src/index.tsx (hunk 5 above)
Comment (andersonmmi) [reply src/index.tsx]: :boom:

File: .gitignore (hunk 1 above)
Comment (rpavlovs) [reply .gitignore]: Also do `git rm -r --cached build`

File: src/index.tsx (hunk 6 above)
Comment (andersonmmi) [reply src/index.tsx]: :sparkles:

**Comment by rpavlovs** [comment]
@andersonmmi remove `/build` folder from git index

**Comment by rpavlovs** [comment]
too late lol, ok, I'll do it in my PR

**Pull request merged**